src/Sulu/Bundle/AdminBundle/Resources/js/components/Masonry/tests/__snapshots__/Masonry.test.js.snap

```diff
@@ -1,41 +1,47 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Render a Masonry container with items 1`] = `
-<div
-  class="masonry"
->
-  <li
-    style="margin-bottom:30px"
+<div>
+  <div
+    class="masonry"
+    style="position: relative; height: 0px;"
   >
-    <div>
-      <img
-        src="http://lorempixel.com/300/200"
-      />
-    </div>
-  </li>
-  <li
-    style="margin-bottom:30px"
-  >
-    <div>
-      <img
-        src="http://lorempixel.com/300/200"
-      />
-    </div>
-  </li>
-  <li
-    style="margin-bottom:30px"
-  >
-    <div>
-      <img
-        src="http://lorempixel.com/300/200"
-      />
-    </div>
-  </li>
+    <li
+      style="margin-bottom: 30px; position: absolute; left: 0px; top: 0px;"
```

Comment:
little bit strange this is added now but looks like react testing library create the snapshot at another timing of component state as previous enzym render method so this change looks fine for me.